web-src/404.html

```diff
@@ -0,0 +1,3 @@
+<!-- <license header> -->
```

Comment:
All these `<license header>` placeholders should be replaced by an Apache license header.

Reply:
This file is removed in the #78 PR